src/com/beltorion/wanderer/repositories/Tile.java

```diff
@@ -0,0 +1,63 @@
+package com.beltorion.wanderer.repositories;
+
+import static com.beltorion.wanderer.services.Images.*;
+
+
+
+import java.awt.*;
+import java.awt.image.BufferedImage;
+
+public class Tile {
+    private TileType type;
+    private int imgPosX;
+    private int imgPosY;
+    //Images images = new Images();
+
+    public Tile() {
+    }
+
+    public Tile(int ImgPosX, int ImgPosY, TileType type) {
+        this.imgPosX = ImgPosX;
+        this.imgPosY = ImgPosY;
+        this.type = type;
+    }
+
+    public void render(Graphics graphics, BufferedImage images, int imgPosX, int imgPosY, TileType type) {
```

Comment:
Ezt nem értem, ha van egy csempéd akkor a kirajzoláshoz miért kell átadni mindent amit amúgy is tud magáról?